src/bot/api.rs

```diff
@@ -237,7 +240,7 @@ impl Bot {
     where
         C: Into<ChatId>,
         Q: Into<String>,
-        O: Into<Vec<String>>
+        O: Into<Vec<String>>,
```

Comment:
Missing `Cow<'a, str>` and `Cow<'a, [String]>`

Reply:
Fixed.